contractcourt/briefcase.go

```diff
@@ -1008,6 +1045,139 @@ func (b *boltArbitratorLog) FetchChainActions() (ChainActionMap, error) {
 	return actionsMap, nil
 }
 
+// LogLocalForceCloseInfo records the passed-in localForceCloseInfo
+// object within the ArbitratorLog for deferred consumption. This is
+// typically invoked when the channel force closure is first initiated
+// either by the user or automatically (link error, HTLC actions, etc.).
+func (b *boltArbitratorLog) LogLocalForceCloseInfo(
+	info localForceCloseInfo) error {
+
+	return kvdb.Update(b.db, func(tx kvdb.RwTx) error {
+		scopeBucket, err := tx.CreateTopLevelBucket(b.scopeKey[:])
+		if err != nil {
+			return err
+		}
+
+		b := new(bytes.Buffer)
+
+		// first byte is userInitiated bool
+		err = binary.Write(b, endian, info.userInitiated)
+		if err != nil {
+			return err
+		}
+
+		// second byte is length of linkFailureError string
+		err = binary.Write(b, endian, uint8(len(info.linkFailureError)))
+		if err != nil {
+			return err
+		}
+
+		// next bytes are the actual linkFailureError string
+		_, err = b.WriteString(info.linkFailureError)
+		if err != nil {
+			return err
+		}
+
+		// next byte is number of key:value pairs in htlcActions map
+		err = binary.Write(b, endian, uint8(len(info.htlcActions)))
+		if err != nil {
+			return err
+		}
+
+		// next bytes are actual map data
```

Comment:
nit: all inline comments should also be full sentences with proper capitalization and punctuation.